Automattic/wp-calypso
# Update/emails add react query to email settings

#### Changes proposed in this Pull Request

This pull request will remove the usage of the state redux pattern and replace it with react query.
For this purpose I have converted the `email-plan.jsx` React component class to a React functional component.

#### Testing instructions

1. Go to emails
2. Check Titan, Forwards, and Goole Workspaces are working and showing all the emails data (mailboxes, warnings)

This Pull Request does not contain any comparison table as it's not applicable in this case. Everything should look and feel the same as before as this changes

Related to {1200182182542585-as-1200213040389307}
